README.md

```diff
@@ -28,13 +28,13 @@ fhir2 --> xds-sender, mpi-client
 
 labintegration --> xds-sender
 
-coreapps --> registrationapp, isanteplus, appointmentschedulingui
+coreapps --> registrationapp, isanteplus, appointmentschedulingui, outgoingmessageexceptions
```

Comment:
Core apps is not a dependency in this module. It's been added to the properties section, but is not being referenced anywhere. Please check.